app/particle/run_md_simulation_3

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# run_md_simulation - run the MD simulation on Compact Cori
+#
+# Author: Nicholas Fong
+#         Lawrence Berkeley National Laboratory
+#         National Energy Research Scientific Computing Center
+#
+# Usage: run_md_simulation [num_tasks] [num_particlecs]
+#
+# Acknowledgment:
+#         This work was supported by the Director, Office of Science,
+#         Division of Mathematical, Information, and Computational
+#         Sciences of the U.S. Department of Energy under contract
+#         DE-AC02-05CH11231.
+#
+#         This research used resources of the National Energy Research
+#         Scientific Computing Center, which is supported by the Office
+#         of Science of the U.S. Department of Energy.
+#
+# Update History:
+# YYYY-MM-DD INITIALS - DESCRIPTION
+# *****************************************************************************
+# 2015-08-12 NF       - Wrote initial script
+
+PROG=$(basename $0)
+
+error() {
+    echo -e "ERROR: $*" >&2
+    exit 1
+}
+
+usage() {
+    echo "USAGE: $PROG [num_tasks] [num_particlecs]"
+
+}
+
+test $# -ne 2 && error "too few arguments" && usage
+
+mpi_host_file='../../mpihostsfile'
+
+# if there is a nonempty string within the brackets, then perform rsync
+# otherwise, do not perform rsync
+if [ "yes" ]; then
```

Comment:
i wanted to have a conditional where it would perform rsync to all the slave nodes or else it would echo 'no rsync', so i thought of using `[` in bash. the string `"yes"` could be any nonempty string, since `[` would be true if any nonempty string is between `[` and `]`. i think i will edit this part of the code so it is more clear, such as creating a boolean instead!